policy/modules/admin/usbguard.te

```diff
@@ -0,0 +1,58 @@
+policy_module(usbguard, 1.0)
+
+########################################
+#
+# Declarations
+#
+
+## <desc>
+##	<p>
+##	Determine whether usbguard-daemon can modify its rules in
+##	/etc/usbguard.
+##	</p>
+## </desc>
+gen_tunable(usbguard_modify_rule_files, false)
```

Comment:
not sure if this is worth a boolean

Reply:
Allowing a daemon to write its own config files does not seem a good idea to me, but this is the default. I'd imagine that in more security conscious environments (where this tool would be also more useful than single user desktops) this would not be allowed. Could there be some other way than using a boolean?

Reply:
What are the scenarios that it would be modifying it's own config files? Can you partition config files into RO and RW files (and corresponding types).